playwright/src/test/java/com/microsoft/playwright/TestPlaywrightCurtomFixturesChannel.java

```diff
@@ -0,0 +1,49 @@
+package com.microsoft.playwright;
+
+import com.microsoft.playwright.junit.Options;
+import com.microsoft.playwright.junit.OptionsFactory;
+import com.microsoft.playwright.junit.UsePlaywright;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIf;
+
+import static com.microsoft.playwright.ServerLifecycle.serverMap;
+import static org.junit.jupiter.api.Assertions.*;
+
+@FixtureTest
+@UsePlaywright(TestPlaywrightCurtomFixturesChannel.CustomOptions.class)
+@EnabledIf("isChannelSpecified")
+public class TestPlaywrightCurtomFixturesChannel {
```

Comment:
I guess it's a typo?

```suggestion
public class TestPlaywrightCustomFixturesChannel {
```